sources/System/Renderer/RenderScreenSystem.hpp

```diff
@@ -0,0 +1,50 @@
+//
+// Created by cbihan on 26/05/2021.
+//
+
+#pragma once
+
+#include "System/System.hpp"
+#include "Camera/Camera2D.hpp"
+#include "Window.hpp"
+
+namespace BBM
+{
+	template <class  T>
```

Comment:
Why is this templated?

Reply:
To allow multiple types of cameras
here 3D and 2D (2D doesn't support it due to missing functions of the wrapper)

Reply:
You can't use both systems at once so templating this system is missleading. You can put a boolean 2D or 3D if you want both or something like that but templating it seems like a bad idea.